packages/attention/src/component.tsx

```diff
@@ -91,8 +114,8 @@ export function Attention(props: AttentionProps) {
   return (
     <div
       aria-describedby={ariaDescribedby}
-      role={ariaRole}
-      aria-label={ariaLabel}
+      role={props.tooltip ? 'tooltip' : 'img'}
+      aria-label={`${activeAttentionProp()} speech bubble pointing ${pointingAt()}`}
```

Comment:
Just occurred to me that this needs to be translated 🙈

Reply:
Oh no you're right! 🙈

Reply:
@BalbinaK I have now updated the PR with handling translation for the aria-label. 😄  It would be nice if we wanted to add the functions that I have commented with a `"TODO"` could maybe be moved to the core-repo, but then we would have to handle translations in the core-repo and not sure if we want to do that? 🤔